source/_patterns/01-elements/buttons/_buttons.md

```diff
@@ -19,11 +19,21 @@ You could set the different button variants via the `data-variant` attribute.
 
 #### SCSS (recommended)
 
-Extend one of the size SCSS placeholders in case that you don't want the default (regular): `%size-Small` or `%size-Large`
+Extend the size SCSS placeholder in case that you don't want the default (medium): `%size-Small`
 
 #### HTML attribute
 
-You could set the different button sizes via the `data-size` attribute, the default is "regular" and in that case is optional.
+You could set the different button sizes via the `data-size` attribute, the default is "medium" and in that case is optional.
+
+### Width
+
+#### SCSS (recommended)
+
+Extend the width placeholders in case that you don't want the default (content-length sized): `%width-Full`
```

Comment:
shouldn't we rename this in %width-full (lowercase)

Reply:
fixed